fauxton-visual-guide/index.html

```diff
@@ -0,0 +1,206 @@
+<p><strong> 4 Jump to a specific document in this database  </strong> 
+Use this search bar to jump to a specific document in the database. This 
+field takes the _id of the document you are trying to view. </p>
+<p><strong> 5 Query Options   </strong>
+You choose how you want to view the documents in this database using 
+Query Options. &#39;Include Docs&#39; will show all the data in each document.
+&#39;Keys&#39; will allow to narrow down the documents being viewed on this 
+page based on the parameters you provide.</p>
+<p><strong> 6 API URL  </strong>
+The API URL button shows up on most pages. All the data in CouchDB is 
+available through a pingable API URL. You can click on this link to view
+and copy the particular API for the page you are viewing.
+Many times, the browser will shows a interface based on one of the API
+endpoints.</p>
+</div></div></div><div id="page-active-tasks" class="pagecontainer"><div class="leftside page"><img src="assets/active-tasks.png"></div><div class="rightside linen page"><div class="textbox"><h1 id="active-tasks">Active Tasks</h1>
+<p><strong> 1 Change Polling Interval  </strong>
+The status of each task can change as you are watching the Active Tasks
+page. The page auto-updates, but polls at regular intervals. You can set
+the amount of time that the page updates itself by dragging on this bar.</p>
+<p><strong> 2 API URL  </strong>
+This page is an interface to the <a href="http://docs.couchdb.org/en/latest/api/server/common.html#api-server-active-tasks">Active Tasks API</a>
+call. You can view or copy the raw JSON by clicking on this button.</p>
+<p><strong> 3 Filter  </strong><br>You can filter by the type of task, or search active tasks by 
+database name (either source or target) using these tabs.</p>
+</div></div></div><div id="page-config" class="pagecontainer"><div class="leftside page"><img src="assets/config.png"></div><div class="rightside linen page"><div class="textbox"><h1 id="configuration">Configuration</h1>
+<p>An interface into the configuration of your CouchDB installation. The 
+interface allows you to edit the different configurable parameters. For 
+more details on configuration, see the 
+<a href="http://docs.couchdb.org/en/latest/config/index.html#config">Configuring CouchDB</a> 
+section. </p>
+</div></div></div><div id="page-cors" class="pagecontainer"><div class="leftside page"><img src="assets/cors.png"></div><div class="rightside linen page"><div class="textbox"><h1 id="cors">CORS</h1>
+<p>CORS stands for Cross Origin Resource Sharing. It is setting that 
+pertains only to browsers.</p>
+<p>By enabling CORS, you allow your browser to access data from other 
+people, recieving from and sending the data to <i>their</i> computers. 
+For example, you can ping someone else&#39;s CouchDB instance through an 
+AJAX request, and it will return information. Or you can POST data from 
+your browser or site, and enabling CORS allows you to send that 
+information.</p>
```

Comment:
maybe this

By enabling CORS, you allow requests that the browser sends from another domain to access/modify the data of your CouchDB. For example, if your CouchDB installation is running on the domain alice.example you can ping someone else's CouchDB instance through an AJAX request from a different domain, e.g. bob.example - or even localhost!